plugin.js

```diff
@@ -88,9 +88,15 @@ class UibookPlugin {
       if (Array.isArray(compilation.entrypoints)) {
         entrypoints = compilation.entrypoints
       } else if (typeof compilation.entrypoints === 'object') {
-        compilation.entrypoints.forEach(function (entrypoint) {
-          entrypoints.push(entrypoint)
-        })
+        if (typeof compilation.entrypoints.forEach === 'function') {
+          compilation.entrypoints.forEach(function (entrypoint) {
+            entrypoints.push(entrypoint)
+          })
+        } else {
+          Object.keys(compilation.entrypoints).forEach((ep) => {
```

Comment:
Could you please replace arrow function with a regular function expression? I didn't use arrow functions in this file, and looks like I forgot to add such a rule to the linter.

```suggestion
          Object.keys(compilation.entrypoints).forEach(function (entrypoint) {
```

Reply:
> Could you please replace arrow function with a regular function expression? I didn't use arrow functions in this file, and looks like I forgot to add such a rule to the linter.

updated